otri/filtering/filters/nuplicator_filter.py

```diff
@@ -1,31 +1,49 @@
-from ..filter import Filter, Stream, Collection
+from ..filter import Filter, Stream, Sequence, Mapping, Any
 import copy
 
 
 class NUplicatorFilter(Filter):
     '''
     N-uplicates the input stream. Placing a copy of the input in each output filter.
-    Inputs: a single Stream.
-    Outputs: Any number of streams.
+
+    Input: 
+        Single stream.
+    Outputs:
+        Any number of streams.
     '''
 
-    def __init__(self, source_stream: Stream, output_streams_count: int, deep_copy: bool = True):
+    def __init__(self, input: str, output: Sequence[str], deep_copy: bool = True):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'input'](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5649486)